README.md

```diff
@@ -1,53 +1,56 @@
 # Obsidian Navigation for VS Code
 
-Navigate Your Obsidian Notes Directly in VS Code!
-This extension allows you to quickly jump between your Obsidian-style Markdown notes (`[[FileName]]`) directly from the text editor in VS Code. If you're an Obsidian user or just prefer organizing your Markdown notes with wiki-style links, this extension will streamline your workflow by letting you open linked Markdown files in your workspace with a single click.
+This extension enhances navigation and file management for Obsidian-like vaults in Visual Studio Code.
```

Comment:
I should probably be less aggressive in the rewrite.

Reply:
of course. Would it make sense to open with a PR that adds the files I think were missing (esbuild.js, some of the .vscode stuff)?